### PR TITLE
Model of Minimal risk rate

### PR DESCRIPTION
**Pull request by leafyoung**
Implement the minimal risk model as described in https://docs.credmark.com/smart-money-in-defi/investment-concepts/minimum-risk-rate-of-defi
- Update Compound pool model with borrow and supply rate in APY. They are equivalent to Aave's rates.
- What's next? To combine some DTOs in Compound and Aave's

### CODE REVIEW

File: models/credmark/protocols/lending/compound/compound_v2.py [hunk 1]
```diff
@@ -295,6 +300,11 @@ class CompoundV2GetPoolInfo(Model):
         }
     }
 
+    # APY calculation
+    ETH_MANTISSA = 1e18
```
Comment (newtectonics) [inline models/credmark/protocols/lending/compound/compound_v2.py:304]: is this decimals?

File: models/credmark/algorithms/minimum_risk.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                description='Rates from stablecoins\' loans to Aave and Compound, '
+                            'then weighted by their debt size and total supply',
+                input=EmptyInput,
+                output=dict)
+class Minrisk(Model):
+    """
+    Doc is
+        https://docs.credmark.com/smart-money-in-defi/investment-concepts/minimum-risk-rate-of-defi
+    """
+    STABLECOINS = {
```
Comment (newtectonics) [inline models/credmark/algorithms/minimum_risk.py:20]: we'll need a model to get the "stablecoins on a chain"

**Review by newtectonics — APPROVED** [review]

File: models/credmark/protocols/lending/compound/compound_v2.py (hunk 1 above)
Comment (leafyoung) [reply models/credmark/protocols/lending/compound/compound_v2.py:304]: According to https://compound.finance/docs. It's an integer.
image

File: models/credmark/algorithms/minimum_risk.py (hunk 2 above)
Comment (leafyoung) [reply models/credmark/algorithms/minimum_risk.py]: ok. I added token.stablecoins.

Comment (newtectonics) [reply models/credmark/algorithms/minimum_risk.py]: yay!

File: models/credmark/tokens/stablecoin.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+@Model.describe(slug='token.stablecoins',
+                version='1.0',
+                display_name='Token - get list of stable coins on a chain',
+                description='A list of stable coins',
+                input=EmptyInput,
+                output=Tokens)
+class StableCoins(Model):
+    STABLECOINS = {
```
Comment (newtectonics) [inline models/credmark/tokens/stablecoin.py:21]: awesome!

File: requirements.txt [hunk 4]
```diff
@@ -4,7 +4,7 @@ numpy>=1.22.1
 matplotlib>=3.5.1
 pandas>=1.4.0
 scipy>=1.8.0
-statsmodels>=0.13.2
```
Comment (newtectonics) [inline requirements.txt:7]: we downgraded this?

**Review by newtectonics — APPROVED** [review]

**Pull request merged**